MaterialLibrary/src/main/java/io/doist/material/elevation/CompatElevationDelegate.java

```diff
@@ -95,7 +95,7 @@ public float getElevation() {
      * @see View#setElevation(float)
      */
     public void setElevation(float elevation) {
-        boolean needsWrap = elevation > mElevation;
+        boolean needsWrap = elevation != mElevation;
```

Comment:
Aha, indeed `0` is a special case! Should we perhaps keep it as `elevation > mElevation || elevation == 0`? We shouldn't need to re-wrap when it decreases (but it's above 0).